app.js

```
@@ -41,103 +31,12 @@
 const SERVER_PORT = parseInt(process.env.SERVER_PORT || 8765);
 
 const server = require('http').createServer(app);
-const io = socketIO(server);
-
+const io = setupSocketIO(server, nekoStateMachine);
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)